settings/.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+	root: true,
```

Comment:
What's this change for?

Reply:
It's to make sure eslint chooses this repo's file, rather than a fallback one higher up the directory structure. The commit message (b107bcfda6a186093b162702d453942c3faa714c) has a more detailed explanation.